### PR TITLE
New feature :  Application Market details

### PR DESCRIPTION
**Pull request by zsl1549**

### CODE REVIEW

File: src/pages/EnterpriseSetting/infrastructure.js [hunk 1]
```diff
@@ -143,6 +143,9 @@ class Infrastructure extends PureComponent {
     } = this.props;
     const { ObjectStorageValue, isEnableObjectStorage } = this.state;
     const params = value || ObjectStorageValue || {};
+    console.log('params',params)
```
Comment (barnettZQG) [inline src/pages/EnterpriseSetting/infrastructure.js:146]: remove log.

File: src/services/market.js [hunk 2]
```diff
@@ -8,41 +8,64 @@ import request from '../utils/request';
...
 
     {
-      method: 'get',
+      method: 'get'
+    }
+  );
+}
+export async function getAppModelsDetails(param) {
+  return request(
+    // `https://doc.goodrain.org/mock/18/console/enterprise/${param.enterprise_id}/app-model/${param.appId}`,
```
Comment (barnettZQG) [inline src/services/market.js:20]: remove debug code

File: src/pages/EnterpriseSetting/infrastructure.js (hunk 1 above)
Comment (zsl1549) [reply src/pages/EnterpriseSetting/infrastructure.js]: Have to repair

File: src/services/market.js (hunk 2 above)
Comment (zsl1549) [reply src/services/market.js]: Have to repair

**Review by barnettZQG — APPROVED** [review]

**Pull request merged**